packages/react/CHANGELOG.md

```diff
@@ -1337,3 +1337,312 @@
 - [#92](https://github.com/primer/brand/pull/92) [`fcef134`](https://github.com/primer/brand/commit/fcef1344f7c1af85691f931851a4dd98a0004099) Thanks [@rezrah](https://github.com/rezrah)! - No updates in this release.
 
   New package versions published to verify the release process.
+
+## 0.5.1
+
```

Comment:
Everything from here has been migrated from the root changelog for historical durability.